src/oop/static_modifier/medium/q5/QuestionText.md

```diff
@@ -0,0 +1,35 @@
+# oop-static_modifier-medium-q5
+
+- Question Type: Java
+
+## Question Text
+
+As the manager of a local apparel store, you are tasked with modelling your store's selection. Make a Java program
+called **Hat**  to model the hats you have in the store using the requirements below:
+
+### Hat Class
+
+Your `Hat` class already the following items:
+
+- Attributes:
+    - A `String` attributes called **colour** that cannot be accessed in other classes
+
+- Constructors:
+    - A constructor that accepts a `String` and initializes the **colour** attribute.
+    - This constructor is accessible from other classes.
+
+You need to add the following items:
+
+- Attributes:
+    - A `String` attribute called **companyName** that has its value, initially `Nike`, shared by all **Hat** instances.
+    - Note that all attributes in this class should be private
```

Comment:
Just add to the attribute description that it should not be visible to other classes. Having this note doesn't really make sense when there is only one attribute they are adding.